packages/react-ui-components/src/components/SchemaForm/SchemaForm.module.scss

```
@@ -1,8 +1,17 @@
 @import "node_modules/@kelvininc/ui-components/dist/assets/styles/globals.scss";
 
 .form-container {
+	/**
+	* @prop --schema-form-x-padding: horizontal form padding
+	* @prop --schema-form-y-padding: vertical form padding
+	* @prop --schema-form-fields-x-gap: horizontal gap between fields
+	* @prop --schema-form-fields-y-gap: vertical gap between fields
+	*/
 	--schema-form-x-padding: 0;
 	--schema-form-y-padding: 0;
+	--schema-form-fields-x-gap: #{$spacing-5x};
+	--schema-form-fields-y-gap: #{$spacing-5x};
```

I don't think you need the curly brackets here, but please confirm it.

```suggestion
	--schema-form-fields-x-gap: $spacing-5x;
	--schema-form-fields-y-gap: $spacing-5x;
```

---

It is necessary because we are defining a css var and we need to define it with value not reference to value